src/main/java/life/qbic/subscriptions/SubscriptionController.java

```diff
@@ -45,13 +48,38 @@
   SubscriptionController(
       SubscriptionRepository subscriptionRepository,
       RequestDecrypter requestDecrypter,
-      RequestEncrypter requestEncrypter
-  ) {
+      RequestEncrypter requestEncrypter) {
     this.subscriptionRepository = subscriptionRepository;
     this.requestDecrypter = requestDecrypter;
     this.requestEncrypter = requestEncrypter;
   }
 
+  @Operation(summary = "Cancel a subscription",
+      parameters = {
+          @Parameter(name = "token",
+              description = "The token of an encrypted cancel request.",
+              example = "For_lfbnS9iTi4Nmwnei4LA_f8SHga1Rdz4yw6aT8zz0V8PaHm1QEbKQTv1jGCEA", schema = @Schema(implementation = String.class))
+      },
+      responses = {
+          @ApiResponse(responseCode = "204", description = "No content. Subscription cancelled, token accepted.",
```

Comment:
NIT:
```suggestion
          @ApiResponse(responseCode = "204", description = "No content. Your subscription was cancelled.",
```
I would try to keep the answers consistent (see the other descriptions, which I find more readable)